lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -1013,6 +1108,146 @@ def __attach_nics(self, entity):
                 self.changed = True
 
 
+def _get_role_mappings(module):
+    if module.params['role_mappings'] is None:
+        return None
+
+    roleMappings = list()
+
+    for roleMapping in module.params['role_mappings']:
+        roleMappings.append(
+            otypes.RegistrationRoleMapping(
+                from_=otypes.Role(
+                    name=roleMapping.params['source'][0]['name'],
```

Comment:
why always using `[0]` here?

Reply:
I did a mistake as you mentioned earlier and used this pattern:
      - source:
          - name: test
That is why it was mapped like a list , now I changed it to be as you suggested so I removed it